### PR TITLE
fixes #8032 test(nimbus): Moves the browser actions of the pref-flip integration test to a specific file

### LINKED ISSUE #8032: Clean up Pref flip integration tests
Since we will be using this config more often, lets move the browser actions to a POM file.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-2982)

### PR DESCRIPTION
**Pull request by jrbenny35**
Because:
* The pref flip integration test included browser steps directly in the test

This Commit:
* Moves them so they're more usable in other tests

### CODE REVIEW

**Review by jaredlockhart — APPROVED** [review]
Mwahahhaha it's beautiful 🤩 🤩 🤩

File: app/tests/integration/nimbus/pages/browser/__init__.py [hunk 1]
```diff
@@ -8,3 +13,33 @@ def execute_script(selenium, *args, script=None, context=None):
...
+    def wait_for_page_to_load(self):
+        self.wait.until(EC.presence_of_element_located(self._search_bar_locator))
+        return self
+
+    def wait_for_pref_flip(self, pref=None):
+        timeout = time.time() + 60 * 5
+        while time.time() < timeout:
+            try:
+                search_bar = self.find_element(*self._search_bar_locator)
+                search_bar.send_keys("nimbus.qa.pref-1")
```
Comment (jaredlockhart) [inline app/tests/integration/nimbus/pages/browser/__init__.py:37]: Oh maybe pull this out into a constant or an optional parameter with a default value or something just so it's clear which pref it's talking about?

Comment (jrbenny35) [reply app/tests/integration/nimbus/pages/browser/__init__.py:37]: ah yes, forgot about that one!

File: app/tests/integration/nimbus/pages/browser/__init__.py [hunk 2]
```diff
@@ -8,3 +13,33 @@ def execute_script(selenium, *args, script=None, context=None):
...
+    _row_locator = (By.CSS_SELECTOR, "tr > td > span > span")
+
+    def __init__(self, selenium, **kwargs):
+        super().__init__(selenium, timeout=80, **kwargs)
+
+    def wait_for_page_to_load(self):
+        self.wait.until(EC.presence_of_element_located(self._search_bar_locator))
+        return self
+
+    def wait_for_pref_flip(self, pref=None, pref_value=None):
```
[the remark this answers is not in the document]
Comment (jrbenny35) [reply app/tests/integration/nimbus/pages/browser/__init__.py:32]: Yeah fair points. It needs both so it shouldn't be allowed to only give one

**Review by jaredlockhart — APPROVED** [review]
Double approve!! 🦅  🦅

**Pull request merged**